app.js

```diff
@@ -1,157 +1,209 @@
+//Wrapped into a function
+function questionFive(){
+  // Change user's input for the fifth question to all lower case.
+  // Validate that their answer matches my answer or not.
+  var answerFive = prompt('Am I a people person?').toLowerCase();
+  console.log('The user answer to question five is: ' + answerFive);
+  // User's input for correct answer can be y or yes.
+  // Depending on true or False, provide feedback.
+  if(answerFive === 'y' || answerFive === 'yes') {
+    alert('You are correct. I enjoy talking to people!');
+    correctAnswers.push(5);
+  } else {
+    // User's input for correct answer can be n or no.
+    (answerFive === 'n' || answerFive === 'no');
+    alert('Oh no, that\'s incorrect. I do enjoy talking to people!');
+  }
 }
+//Calling questionFive function
+questionFive();
 
 // Question six.
-// Change user's input for the six question to all lower case.
-var answerSix = prompt('Can you guess what\'s my favorite food?');
-console.log('The answer to question five is: ' + answerSix);
-// Create an array and assign it to a variable.
-var food = ['grilled salmon', 'fried chicken', 'philly cheesesteak', 'pasta'];
-food.indexOf('grilled salmon');
-food.indexOf('fried chicken');
-food.indexOf('philly cheesesteak');
-food.indexOf('pasta');
-// Use the indexOf method to point out the correct answer in the array.
-if(food.indexOf('grilled salmon') === 0) {
-  alert('I love it too however my favorite is Grilled Salmon.');
-  correctAnswers.push(6);
-} else {
-  alert('My favorite is Grilled Salmon but that\'s good too');
+//Wrapped into a function
+function questionSix(){
+  // Change user's input for the six question to all lower case.
+  var answerSix = prompt('Can you guess what\'s my favorite food?').toLowerCase();
+  console.log('The user answer to question six is: ' + answerSix);
+  // Create an array and assign it to a variable.
+  var food = ['grilled salmon', 'fried chicken', 'philly cheesesteak', 'pasta'];
+  // food.indexOf('grilled salmon');
+  // food.indexOf('fried chicken');
+  // food.indexOf('philly cheesesteak');
+  // food.indexOf('pasta');
+  // Use the indexOf method to point out the correct answer in the array.
+  if(food.indexOf(answerSix) >= 0) {
+    alert('That is correct!');
+    correctAnswers.push(6);
+  } else {
+    alert('My favorite is ' + food + ', but that\'s good too');
+  }
 }
+//Calling questionSix function
+questionSix();
 
 // Question seven.
-// Use a do while loop to execute code to loop through a series of if else statments.
-// Start the loop at 0.
-var counter = 0;
-do {
-  counter++;
-  var answerSeven = prompt('Knowing that I was in the military, can you guess how many times I\'ve moved in the past?');
-  console.log('The answer to question seven is: ' + answerSeven);
+//Wrapped into a function
+function questionSeven() {
+  // Use a do while loop to execute code to loop through a series of if else statments.
+  // Start the loop at 0.
+  var counter = 0;
+  do {
```

Comment:
Good use of a do while loop. Not something I see very often, so it's cool to see you implement it here.

It's most a good pattern for readability, and it works great here. :)